stdlib/ballerina-database/src/main/ballerina/sql/Package.md

```diff
@@ -0,0 +1,21 @@
+## Package overview
+
+This package provides the common record types and constants required for other data management packages such as jdbc, mysql, h2. 
+
+### PoolOptions 
+
+The PoolOptions type the properties which are used to configure DB connection pool. This is used with jdbc, mysql and h2 endpoints to configure the connection pool associated with the endpoint.
```

Comment:
Change "which" to "that", add back ticks around jdbc, mysql, and h2, add a comma after mysql